services/redis.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sd-redis
+  labels:
+    name: redis
+    app: sd-redis
+spec:
+  ports:
+  - port: 6379
+    name: master
+  clusterIP: None
+  selector:
+    name: redis
+---
+apiVersion: apps/v1beta1
```

Comment:
It's a lot more meaningful if we just state that this is a config for Kubernetes >= 1.6 and not even mention legacy versions